src/providers/v2/pool-provider.ts

```diff
@@ -144,7 +164,35 @@ export class V2PoolProvider implements IV2PoolProvider {
         continue;
       }
 
-      const [token0, token1] = sortedTokenPairs[i]!;
+      let [token0, token1] = sortedTokenPairs[i]!;
+      if (
+        tokenPropertiesMap[token0.address.toLowerCase()]
+          ?.tokenValidationResult === TokenValidationResult.FOT
+      ) {
+        // TODO add buyFeeBps and sellFeeBps once we upgrade sdk-core
```

Comment:
we should update this now that sdk is launched right?

Reply:
Yep, updated to pass in the FOT taxes.

Reply:
remove the todo btw